src/config.h

```diff
@@ -77,7 +77,7 @@ struct config {
 		try {
 			std::string mkey(key);
 			std::vector<char> v((char *)&value,
-					    (char *)&value + sizeof(value));
+					    (char *)&value + sizeof(void *));
```

Comment:
this doesn't fix anything and is actually backward - using variables in sizeof is preferred